src/main/java/vg/civcraft/mc/civmodcore/serialization/NBTSerialization.java

```diff
@@ -0,0 +1,160 @@
+package vg.civcraft.mc.civmodcore.serialization;
+
+import com.google.common.base.Strings;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Modifier;
+import java.util.HashMap;
+import java.util.Map;
+import vg.civcraft.mc.civmodcore.api.NBTCompound;
+
+/**
+ *
+ * */
+public final class NBTSerialization {
+
+	private static final String NBT_CLASS_PATH_KEY = "==";
+    private static final Map<String, Class<? extends NBTSerializable>> registeredClasses = new HashMap<>();
```

Comment:
weird indentation here

Reply:
Weird, might need to re-apply the editorconfig to this file